### DISCUSSION TITLE
ESP32 with CC2530+CC2591

### DISCUSSION THREAD

**Discussion by mantielero** [arendst/Tasmota#19254] [Q&A]
I was wondering if they can play well together. I was trying connecting it as I do with a Cc2530 and the permit to join appear greyout.

**Comment by sfromis** [comment]
Did you flash it with a Tasmota-compatible firmware build? To see what happens, watch the console (after having activated `Weblog 3`).

**Comment by mantielero** [comment]
I flashed with my own compilation. After running it with the official one, it worked.
